spk/exfat/src/installer.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Package
+PACKAGE="exfat"
+DNAME="exFAT"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put fuse-exfat & exfat-utils in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/sbin/mount.exfat-fuse /usr/local/bin/mount.exfat
```

Comment:
Seems this should be changed to `ln -s ${INSTALL_DIR}/sbin/mount.exfat /usr/local/bin/mount.exfat`, right?